packages/cashier/src/Components/side-note.jsx

```diff
@@ -1,39 +1,80 @@
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import React from 'react';
 import { DesktopWrapper, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { isMobile } from '@deriv/shared';
 import 'Sass/side-note.scss';
 
-const SideNoteBullet = ({ children }) => (
+const SideNoteTitle = ({ side_notes, title }) => (
+    <Text className='side-note__title' weight='bold' as='p'>
+        {title ||
+            (side_notes?.length > 1 ? <Localize i18n_default_text='Notes' /> : <Localize i18n_default_text='Note' />)}
+    </Text>
+);
+
+const SideNoteText = ({ children }) => (
+    <Text className='side-note__text' size='xxs' as='p'>
+        {children}
+    </Text>
+);
+
+const SideNoteBullet = ({ children, component }) => (
     <div className='side-note__bullet-wrapper'>
         <div className='side-note__bullet' />
-        <span>{children}</span>
+        {component ? children : <SideNoteText>{children}</SideNoteText>}
     </div>
 );
 
-const SideNote = ({ notes, title, has_bullets }) => (
-    <div className='side-note'>
-        {title && (
-            <DesktopWrapper>
-                <Text className='side-note__text' weight='bold' as='p'>
-                    {title}
-                </Text>
-            </DesktopWrapper>
-        )}
-
-        {notes.map((note, i) =>
-            has_bullets ? (
-                <SideNoteBullet key={i}>{note}</SideNoteBullet>
-            ) : (
-                <Text key={i} className='side-note__text' size='xxs' as='p'>
-                    {note}
-                </Text>
-            )
-        )}
-    </div>
-);
+const SideNote = ({ side_notes, title, has_bullets = true, className }) => {
+    const checkNote = (note, i) => {
```

Comment:
I use `checkNote` because this method is checking the note to decide what is it and how should it be returned, but `notes` is better. I changed that to `notes`.